backend/api/schedule_insert.js

```diff
@@ -0,0 +1,15 @@
+const schedule = require('node-schedule')
+const insert_jobs = require('./insert_jobs')
+
+const rule = new schedule.RecurrenceRule()
+rule.hour = 15 //23
```

Comment:
Sugestões:
=========

- alterar **rule.hour** de 15 para 23 , pois nós como programadores se deixarmos para depois não fazemos kkkkk
- alterar também **rule.minute** de 07 para 30 também.

Alterações e pontos de atenção:
===============================
- Quando finalizar funcionalidade deve abrir um PR para branch **develop**, assim como explico nas instruções do projeto sobre fluxo do git https://github.com/devpleno/junta-vagas#fluxo-de-git-para-trabalhar-no-projeto
- Criar arquivo **.gitignore** na raiz do projeto é colocar o seguinte conteúdo no arquivo: **node_modules/**
- Você comitou os o diretório **node_modules/** isso é desnecessário, pois quando outro dev 
 for trabalhar basta rodar o comando **npm install** que o npm irá baixar o módulos necessários para aplicação rodar. OBS: link para ajudar a remover o node_modules do github  https://medium.com/@andgomes/git-removendo-arquivos-de-um-reposit%C3%B3rio-7eed699a035f